test/Microsoft.TestPlatform.TestUtilities/IntegrationTestEnvironment.cs

```diff
@@ -17,16 +17,22 @@ namespace Microsoft.TestPlatform.TestUtilities
     /// </summary>
     public class IntegrationTestEnvironment
     {
+        public static string TestPlatformRootDirectory = Environment.GetEnvironmentVariable("TP_ROOT_DIR");
+        public static string buildConfiguration =
```

Comment:
Now this class exposes two public members, one instance and static, both provide build configuration. I understand the requirement, may be there's a better way to do it. If there's none, I'd prefer creating a separate private buildConfiguration in new test class.

Reply:
Value BuildConfiguration remains same throughout the lifetime of test. I would prefer to remove the property and use static variable.

Reply:
Okay. Static property then.